### ISSUE TITLE
Val mean AP shows nan during training the detector

### ISSUE THREAD

**Issue by icesohelrana** [phil-bergmann/tracking_wo_bnw#8]
[VAL]: im_detect: 5316/5316 0.107s 0.001s   
[TRAIN]: Mean AP = 0.5192
/tracking_wo_bnw/src/fpn/fpn/datasets/mot.py:420: RuntimeWarning: invalid value encountered in true_divide
  rec = tp / float(npos)

[VAL]: Mean AP = nan

I was trying to train the detector. But it shows the following nan value for test set mean AP of MOT17 dataset.

**Comment by timmeinhardt** [comment]
There is no publicly available ground truth for the test set. In order to validate the successful training, one has to split the training set and do cross-validation.

**Comment by icesohelrana** [comment]
Thanks a lot
